Xamarin.Forms.Platform.UAP/MediaElementRenderer.cs

```diff
@@ -143,16 +148,43 @@ void Control_CurrentStateChanged(object sender, RoutedEventArgs e)
 					break;
 			}
 
-			Element.SendCurrentState((MediaElementState)((int)Control.CurrentState));
+			((IMediaElementController)Element).CurrentState = FromWindowsMediaElementState(Control.CurrentState);
+		}
+
+		static MediaElementState FromWindowsMediaElementState(Windows.UI.Xaml.Media.MediaElementState state)
+		{
+			switch(state)
+			{
+				case Windows.UI.Xaml.Media.MediaElementState.Buffering:
+					return MediaElementState.Buffering;
+
+				case Windows.UI.Xaml.Media.MediaElementState.Closed:
+					return MediaElementState.Closed;
+
+				case Windows.UI.Xaml.Media.MediaElementState.Opening:
+					return MediaElementState.Opening;
+
+				case Windows.UI.Xaml.Media.MediaElementState.Paused:
+					return MediaElementState.Paused;
+
+				case Windows.UI.Xaml.Media.MediaElementState.Playing:
+					return MediaElementState.Playing;
+
+				case Windows.UI.Xaml.Media.MediaElementState.Stopped:
+					return MediaElementState.Stopped;
+			}
+
+			return 0;
```

Comment:
`0` equals `MediaElementState.Closed`. I think it's more correct to throw an exception.